homeassistant/components/template/light.py

```diff
@@ -243,13 +305,15 @@ def supported_features(self):
         """Flag supported features."""
         supported_features = 0
         if self._level_script is not None:
-            supported_features |= SUPPORT_BRIGHTNESS
+            supported_features |= SUPPORT_BRIGHTNESS | SUPPORT_TRANSITION
```

Comment:
Not all lights support transitions, don't we want to make this configurable?

Reply:
done